src/libs/brands.ts

```diff
@@ -24,14 +24,52 @@ import terraLogoShadow from 'src/images/brands/terra/logo-wShadow.svg'
 const nonBreakingHyphen = '\u2011'
 
 
+export interface BrandConfiguration {
+  /** Brand name */
```

Comment:
@msilva-broad and I tried out a few styles of JSDoc comments for this type and found this way to be the best at surfacing information in VS Code.